Cargo.toml

```diff
@@ -175,81 +175,137 @@ aptos-build-info = { path = "crates/aptos-build-info" }
 netcore = { path = "network/netcore" }
 network = { path = "network" }
+network-builder = { path = "network/builder" }
+safety-rules = { path = "consensus/safety-rules" }
 schemadb = { path = "storage/schemadb" }
 scratchpad = { path = "storage/scratchpad" }
 short-hex-str = { path = "crates/short-hex-str" }
+state-sync-driver = { path = "state-sync/state-sync-v2/state-sync-driver" }
 storage-interface = { path = "storage/storage-interface" }
 storage-service-client = { path = "state-sync/storage-service/client" }
 storage-service-server = { path = "state-sync/storage-service/server" }
 storage-service-types = { path = "state-sync/storage-service/types" }
+transaction-emitter-lib = { path = "crates/transaction-emitter-lib" }
 vm-genesis = { path = "aptos-move/vm-genesis" }
 vm-validator = { path = "vm-validator" }
 
 # External crate dependencies.
 # Please do not add any test features here: they should be declared by the individual crate.
-anyhow = "1.0.57"
+anyhow = "1.0.62"
+async-stream = "0.3"
 async-trait = "0.1.53"
+base64 = "0.13.0"
 bcs = { git = "https://github.com/aptos-labs/bcs", rev = "2cde3e8446c460cb17b0c1d6bac7e27e964ac169" }
+bigdecimal = { version = "0.3.0", features = ["serde"] }
+byteorder = "1.4.3"
 bytes = "1.1.0"
+chrono = { version = "0.4.19", features = ["clock", "serde"] }
 claims = "0.7"
-clap = "3.1.17"
+clap = "3.2.17"
+const_format = "0.2.26"
+criterion = "0.3.5"
+diesel = { version = "1.4.8", features = ["chrono", "postgres", "r2d2", "numeric", "serde_json"] }
+diesel_migrations = { version = "1.4.0", features = ["postgres"] }
 enum_dispatch = "0.3.8"
+env_logger = "0.9.0"
 fail = "0.5.0"
-futures = "0.3.21"
+field_count = "0.1.1"
+futures = "= 0.3.24" # Previously futures v0.3.23 caused some consensus network_tests to fail. We now pin the dependency to v0.3.24.
+futures-channel = "= 0.3.24"
+gcp-bigquery-client = "0.13.0"
+get_if_addrs = "0.5.3"
 goldenfile = "1.1.0"
 hex = "0.4.3"
-hyper = "0.14.18"
+http = "0.2.3"
+hyper = { version = "0.14.18", features = ["full"] }
 indoc = "1.0.6"
 itertools = "0.10.3"
+jemallocator = { version = "0.3.2", features = ["profiling", "unprefixed_malloc_on_supported_platforms"] }
+log = "0.4.17"
 lru = "0.7.5"
 maplit = "1.0.2"
 mime = "0.3.16"
+mirai-annotations = "1.12.0"
 mockall = "0.11.0"
+num-derive = "0.3.3"
 num-traits = "0.2.15"
 once_cell = "1.10.0"
 paste = "1.0.7"
+pbjson = "0.4.0"
 percent-encoding = "2.1.0"
 poem = { version = "1.3.40", features = ["anyhow", "rustls"] }
-poem-openapi = { version = "2.0.10", features = ["url"] }
+poem-openapi = { version = "2.0.10", features = ["swagger-ui", "url"] }
+prometheus-parse = "0.2.2"
 proptest = "1.0.0"
+prost = "0.10.4"
+prost-types = "0.10.1"
 rand = "0.7.3"
+rayon = "1.5.2"
 regex = "1.5.5"
-reqwest = { version = "0.11.10", features = ["blocking", "json"] }
-serde = { version = "1.0.137", features = ["derive"] }
+reqwest = { version = "0.11.11", features = ["blocking", "cookies", "json"] }
```

Comment:
Changing version can have unexpected consequences as seen with tokio, we're sure nothing's going to change with all of these updates?

Reply:
I'd hope not. But if it does, it's better to force it now and unify to a single version than continue to depend on multiple versions and face much tougher problems later on. 😄 FWIW, the only thing that changed in this PR was the we removed the dependency on `"env_logger 0.8.4"` and just settle at `"env_logger 0.9.0",`